manifests/init.pp

```diff
@@ -69,27 +69,18 @@
 # limitations under the License.
 #
 class openvpn (
-  $autostart_all                   = true,
-  $manage_service                  = true,
-  $client_defaults                 = hiera_hash('openvpn::client_defaults', {}),
-  $clients                         = hiera_hash('openvpn::clients', {}),
-  $client_specific_config_defaults = hiera_hash('openvpn::client_specific_config_defaults', {}),
-  $client_specific_configs         = hiera_hash('openvpn::client_specific_configs', {}),
-  $revoke_defaults                 = hiera_hash('openvpn::revoke_defaults', {}),
-  $revokes                         = hiera_hash('openvpn::revokes', {}),
-  $server_defaults                 = hiera_hash('openvpn::server_defaults', {}),
-  $servers                         = hiera_hash('openvpn::servers', {}),
+  $autostart_all                        = true,
+  $manage_service                       = true,
+  Hash $client_defaults                 = hiera_hash('openvpn::client_defaults', {}),
```

Comment:
What's the added benefit of ```hiera_hash``` here?

Reply:
I've no idea so I didn't want to touch it. Is this for strange hiera lookups? I guess this is obsolete, but should be addressed in a separate pull request.

Reply:
I think it's redundant but agree it's fine to do it in another PR (hence my approval rather than a comment/request).